kedro_mlflow/framework/hooks/node_hook.py

```diff
@@ -51,6 +90,9 @@ def before_node_run(
         mlflow.log_params(params_inputs)
 
 
+mlflow_node_hooks = MlflowNodeHook()
```

Comment:
Can you remove this line? It does not belong to this PR, it is linked to the auto-regsitration one.

Reply:
It's not just for auto-registration, it's also for declaring hooks in .kedro.yml that we normally support since we moved the hook's configurations into `mlflow.yml`. When the `_register_hooks` is collecting hooks in .kedro.yml or in entry_points, he don't instantiate the hook object, they use our class as is, which raise an error when kedro use the hook, because he don't get the `self` argument.
So it's more a fix for precedent release. We can let this fix for the auto-regsitration PR

Reply:
I don't really understand why Kedro would throw an error, and I am not aware of this bug in previous release (I mean that no user gave me such feedback, but I only try it with `kedro>=0.16.0,<0.16.4`). Is it linked to kedro 0.16.5 new hook management? If yes, let's add it to the PR to fix #62 then.

Reply:
Oh ! I thought declaring hooks in .kedro.yml was possible on kedro <0.16.5. It's just introduced on 0.16.5
So yeah, we can throw it to #62